pkg/settings/setting.go

```diff
@@ -437,3 +437,17 @@ func GetRancherVersion() string {
 	}
 	return strings.TrimPrefix(rancherVersion, "v")
 }
+
+// IterWhitelistedEnvVars iterates over the environment variables whitelisted by CATTLE_WHITELIST_ENVVARS.
+// If the variable is whitelisted but unset or empty, the handler function will not be called.
+func IterWhitelistedEnvVars(handler func(name, value string)) {
```

Comment:
I like this approach. One nit is the function name, I don't see us generally using `Iter` as prefix, so maybe go for full name like `IterateWhitelistedEnvVars` which is not _that_ much longer.

Otherwise LGTM.

Reply:
Sure!